packages/configs/jest/config.js

```diff
@@ -0,0 +1,42 @@
+const nextJest = require('next/jest')
```

Comment:
I debated whether or not to use `next/jest`, but as most of our repos (except this one and `mktg-content-workflows`) have a Next app, it seemed reasonable!

💭 I could move `next` to an optional dependency and create two separate configs, but I think for now this is a very reasonable starting point.